### PR TITLE
steps26-32

### PR DESCRIPTION
**Pull request by Yanetta**

### CODE REVIEW

File: src/main/resources/steps26-32.sql [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+    salesreps
+    INNER JOIN salesreps mgrs ON salesreps.manager = mgrs.empl_num;
+
+SELECT
+    salesreps.name,
+    mgrs.name
+FROM
+    salesreps,
+    salesreps mgrs
+WHERE
```
Comment (mkharkhuta) [inline src/main/resources/steps26-32.sql]: you need  join here

File: src/main/resources/steps26-32.sql [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+    orders      ord
+    INNER JOIN products    pr ON ord.product = pr.product_id
+    INNER JOIN customers   cus ON ord.cust = cus.cust_num
+WHERE
+    pr.description LIKE '%a%'
+    OR pr.description LIKE '%A%'
+    OR pr.description LIKE '%А%'
+    OR pr.description LIKE '%а%';
+
+SELECT DISTINCT
```
Comment (mkharkhuta) [inline src/main/resources/steps26-32.sql:14]: this request is not necessary

File: src/main/resources/steps26-32.sql [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+WHERE
+    manager = empl_num;
+
+SELECT
+    emps.name,
+    mgrs.name
+FROM
+    salesreps   emps,
+    salesreps   mgrs
+WHERE
```
Comment (mkharkhuta) [inline src/main/resources/steps26-32.sql]: you need join here

**Review by mkharkhuta — APPROVED** [review]
very fast and correct error correction! fine!)

File: src/main/resources/steps26-32.sql (hunk 2 above)
Comment (SerhiiPasko) [reply src/main/resources/steps26-32.sql:14]: Yes,repeat same

File: src/main/resources/steps26-32.sql [hunk 4]
```diff
@@ -0,0 +1,314 @@
...
+    order_num,
+    amount,
+    description
+FROM
+    orders     ord
+    INNER JOIN products   pr ON ord.mfr = pr.mfr_id
+                              AND ord.product = pr.product_id;
+
+SELECT
+    order_num,
```
Comment (SerhiiPasko) [inline src/main/resources/steps26-32.sql:71]: If you use alias for table names, use this alias before fields too

**Review by SerhiiPasko — APPROVED** [review]
Read comments

**Pull request merged**